sdk/src/client/api/block_builder/input_selection/requirement/mana.rs

```diff
@@ -15,7 +15,6 @@ impl InputSelection {
                 input.output_id().transaction_id().slot_index(),
                 self.slot_index,
             )?;
-            // TODO rewards https://github.com/iotaledger/iota-sdk/issues/1310
```

Comment:
Why are we not handling rewards here ?

Reply:
there's just nothing needed here, or am I missing something?

Reply:
Shouldn't rewards contribute to meet the required output mana ?

Reply:
They are added to the input total, and that causes them to be put in a remainder

Reply:
But that's a completely different process? Here we are trying to find inputs to match the requirement, if we don't consider rewards we may have to select more inputs than needed?

Reply:
But we don't need to select inputs to get the reward mana, it is provided in a totally different way

Reply:
Yes but the fact that dele delegation are already selected gives you the reward and can prevent you from selecting other inputs that wouldn't have been required

Reply:
fixed

Reply:
danke dir